generators/server/templates/src/main/java/package/web/rest/AccountResource.java.ejs

```diff
@@ -377,7 +377,7 @@ public class AccountResource {
      */
     @PostMapping(path = "/account/change-password")
     public <% if (reactive) { %>Mono<Void><% } else { %>void<% } %> changePassword(@RequestBody PasswordChangeDTO passwordChangeDto) {
-        if (!checkPasswordLength(passwordChangeDto.getNewPassword())) {
+        if (checkPasswordLength(passwordChangeDto.getNewPassword())) {
```

Comment:
I'm not convinced that inverting the meaning of `checkPasswordLength` is an improvement. 
Or propose a new meaningful name for this method as e.g. `isPasswordLengthInvalid`